<launch>

    <node pkg="test_pluto" name="pose_offset_pub" type="pose_offset_pub.py" output="screen">
        <env name="PYTHONPATH" value="$(env PYTHONPATH):/usr/lib/python2.7/dist-packages"/>
        <env name="ROS_PYTHON_VERSION" value="2"/>
    </node>

    <node pkg="test_pluto" name="robot_pose_publisher" type="publishers.py" output="screen">
        <env name="PYTHONPATH" value="$(env PYTHONPATH):/usr/lib/python2.7/dist-packages"/>
        <env name="ROS_PYTHON_VERSION" value="2"/>
    </node>

    <node pkg="test_pluto" name="goal_publisher" type="send_goal.py" output="screen">
        <env name="PYTHONPATH" value="$(env PYTHONPATH):/usr/lib/python2.7/dist-packages"/>
        <env name="ROS_PYTHON_VERSION" value="2"/>
    </node>

    <node pkg="test_pluto" name="pluto_navigation_system" type="navigatecombine5.py" output="screen">
        <env name="PYTHONPATH" value="$(env PYTHONPATH):/usr/lib/python3.6/dist-packages"/>
        <env name="ROS_PYTHON_VERSION" value="3"/>
    </node>

</launch>
